libs/langchain-baidu-qianfan/src/tests/chat_models.int.test.ts

```diff
@@ -5,21 +5,21 @@ import { ChatBaiduQianfan } from "../chat_models.js";
 
 test("invoke", async () => {
   const chat = new ChatBaiduQianfan({
-    model: "ERNIE-Bot-turbo",
+    model: "ERNIE-Lite-8K",
   });
   const message = new HumanMessage("北京天气");
   const res = await chat.invoke([message]);
-  // console.log({ res });
+  console.log(res.content);
```

Comment:
nit, dont commit console.log
```suggestion
  // console.log(res.content);
```